Lib/asyncio/tasks.py

```diff
@@ -931,18 +931,21 @@ def callback():
 
 
 def create_eager_task_factory(custom_task_constructor):
-
-    if "eager_start" not in inspect.signature(custom_task_constructor).parameters:
-        raise TypeError(
-            "Provided constructor does not support eager task execution")
+    """
+    Creates a function suitable for use as a task factory on an event-loop.
+    The tasks created will be started immediately (rather than being first
```

Comment:
If we want to follow PEP 257 (generally a good default unless there are local-consistency reasons not to), there should be a blank line after the one-line summary and before further details (https://peps.python.org/pep-0257/#multi-line-docstrings)

```suggestion
    Creates a function suitable for use as a task factory on an event-loop.
    
    The tasks created will be started immediately (rather than being first
```